docker-compose.yml

```diff
@@ -31,152 +32,113 @@ services:
       - CD_BADGES=badges
       - CD_EVENTBRITE=eventbrite
       - CD_ORGANISATIONS=organisations
+      - NODE_ENV=development
     volumes:
-      - ./workspace-zen/cp-translations:/usr/src/cp-translations:Z
       - ./workspace-zen/cp-zen-platform:/usr/src/app:Z
-      - ./workspace-zen/cp-zen-frontend:/usr/src/cp-zen-frontend:Z
+      - ./workspace-zen/cp-zen-frontend:/usr/src/app/node_modules/cp-zen-frontend:Z
+      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:Z
   frontend:
-    image: coderdojo/cp-zen-frontend
+    image: node:carbon
+    command: yarn dev
+    working_dir: /usr/src/app
+    environment:
+      - NODE_ENV=development
     volumes:
-      - ./workspace-zen/cp-translations:/usr/src/cp-translations:Z
+      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:Z
```

Comment:
isn't that hte role of the image to ln -s over to node_module?
https://github.com/CoderDojo/cp-zen-frontend/blob/master/docker-entrypoint.sh#L11

Reply:
this replaces that and works more reliably than a symlink

Reply:
then update create yourself a ticket to replace the one in the repos, elsewhat both will be done

Reply:
no this doesnt run the ln script
i was goign to remove the ln scripts once this was merged